### PR TITLE
Add algorithm playground

### PR DESCRIPTION
**Pull request by MariMalta**
## What does this PR do?
Add resource

## For resources
Algorithms playgrounds
### Description
It's a playground to learn and train algorithms
### Why is this valuable (or not)?
Because its a new topic for the repo with an playground of algorithm, that it's very valuable in a programmer life

### How do we know it's really free?
It's a free open source playground, in the link we can see that it's free.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-playgrounds.md [hunk 1]
```diff
@@ -53,6 +54,12 @@
 * [Vim](#vim)
 
 
+### Algorithms
+
+* [The Algorithm Visualizer](https://algorithm-visualizer.org)
+* [The Algorithms](https://the-algorithms.com)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:60]: this one isn't a playground. I think it fits better in books.

File: more/free-programming-playgrounds.md [hunk 2]
```diff
@@ -53,6 +54,12 @@
 * [Vim](#vim)
 
 
+### Algorithms
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:57]: there's a resource in interactive playgrounds that should move here.

**Review by eshellman — CHANGES_REQUESTED** [review]
neat resources, sot so easy to categorize!

File: more/free-programming-playgrounds.md (hunk 1 above)
Comment (MariMalta) [reply more/free-programming-playgrounds.md:60]: The one you are talking about is the algorithms (https://the-algorithms.com)? I can move it to books. :)

File: more/free-programming-playgrounds.md (hunk 2 above)
Comment (MariMalta) [reply more/free-programming-playgrounds.md:57]: What resource? Sorry, I don't really understand what you're saying here

Comment (joaoluizn) [reply more/free-programming-playgrounds.md:57]: I think this is the one:
https://github.com/EbookFoundation/free-programming-books/blob/main/more/free-programming-interactive-tutorials-en.md#algorithms

@eshellman

File: more/free-programming-playgrounds.md (hunk 1 above)
Comment (eshellman) [reply more/free-programming-playgrounds.md:60]: yes please

Comment (eshellman) [reply more/free-programming-playgrounds.md:60]:
```suggestion
```
moved to books in #10986

File: more/free-programming-playgrounds.md [hunk 3]
```diff
@@ -53,6 +54,11 @@
...
+
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:58]:
```suggestion

* [Data Structures Visualization](https://www.cs.usfca.edu/~galles/visualization/Algorithms.html) - David Galles
```
moved from interactive tutorials

**Pull request merged**